CHANGELOG

```diff
@@ -408,12 +411,12 @@ v3.8.3 (2021-11-17)
   replication operations.
 
 * Close a potential gap during shard synchronization when moving from the
-  initial sync step to the WAL tailing step. In this small gap the leader 
+  initial sync step to the WAL tailing step. In this small gap the leader
```

Comment:
Can we avoid rewriting the full CHANGELOG file?

Reply:
done